help/markdown/fake-template.md

```diff
@@ -27,21 +27,21 @@ to run your build. Have a look at the [fake commandline](fake-commandline.html)
 Specifies the name of the generated build-script. Defaults to `build.fsx`.
 
 ### --bootstrap
-Specifies your prefered way to bootstrap FAKE.
+Specifies your preferred way to bootstrap FAKE.
 
-- `local` (default) - Installs the FAKE dotnet sdk local tool into the `dotnet-tools.json` manifest. This requires an dotnet SDK version 3 or higher.
+- `local` (default) - Installs the FAKE dotnet sdk local tool into the `dotnet-tools.json` manifest. This requires dotnet SDK version 3 or higher.
 - `tool` - Installs the FAKE dotnet sdk global tool into the `--tool-path` folder
 - `none` - Does not bootstrap FAKE. Use this if you want to use a global installation of FAKE
 
 ### --dependencies
-Specifies your prefered way to define the nuget packages used in your build:
+Specifies your preferred way to define the nuget packages used in your build:
 
 - `file` (default) - Creates a `paket.dependencies` file to define build dependencies
 - `inline` - Defines build dependencies inside the build script
```

Comment:
The issue was quite a while ago but in addition to your change above `inline` should have `(default)` now, correct?

Reply:
Yes you are right. I'll update when I can. I am new to making contribution on Github though. So question is how do I proceed with updating the documentation following your comment? Can I somehow update this pull pull request?

Reply:
> Can I somehow update this pull pull request?

Indeed, just commit and push to your branch, it will be updated automatically.

Reply:
Apologies for being pedantic or maybe it's my lack of understanding but for this line

- `inline` - Defines build dependencies inside the build script

which is now the default, the description, "Defines build dependencies inside the build script", is a bit unclear to me. 

Do you mean that in our build script e.g. build.fsx, 

we would have to do something like:

``` fsharp
#r "paket:
nuget Fake.DotNet.Cli
nuget Fake.IO.FileSystem
nuget Fake.Core.Target
nuget FSharp.Data //"
```
If so, I think including it in the documentation might be useful because I'm not familiar with this construct. 

From my understanding this is a FAKE specific construct because if I try to send this to FSI, I'll get something like 

```
error FS3216: Package manager key 'paket' was not registered in [/usr/share/dotnet/sdk/3.1.201/FSharp; /usr/share/dotnet/sdk/3.1.201/FSharp/], []. Currently registered: nuget
```